server/services/store/sqlstore/boards_and_blocks.go

```diff
@@ -141,6 +141,11 @@ func (s *SQLStore) duplicateBoard(db sq.BaseRunner, boardID string, userID strin
 	if err != nil {
 		return nil, nil, err
 	}
+	// if original board is template
+	// make new board private
+	if board.IsTemplate {
+		board.Type = "P"
```

Comment:
Just a note that we should set all duplicated boards to private, based on the followup from the bugbash this week. I filed a separate bug to track that: https://github.com/mattermost/focalboard/issues/2822.